newsfragments/3334.added.md

```diff
@@ -0,0 +1 @@
+Add `PyErr_Display` for all Python versions, and FFI symbol `PyErr_DisplayException` for Python 3.12.
```

Comment:
Shouldn't this reference `PyErr::display` which was added? (`PyErr_Display` does not seem to be new.)

Reply:
Yes, typo, will force-push fix.